documentation/source/Usecases/couchbase.rst

```diff
@@ -0,0 +1,116 @@
+    default       pvc-16210b06-c7ba-11e7-892e-000c29119159-ctrl-78db5f845b-v7w5s   1/1       Running   0          11h
+    default       pvc-16210b06-c7ba-11e7-892e-000c29119159-rep-94d9844df-78zsm     1/1       Running   0          11h
+    default       pvc-16210b06-c7ba-11e7-892e-000c29119159-rep-94d9844df-rh4xs     1/1       Running   0          11h
+    default       pvc-40e1b64f-c7ba-11e7-892e-000c29119159-ctrl-c54b6969b-75mjj    1/1       Running   0          11h
+    default       pvc-40e1b64f-c7ba-11e7-892e-000c29119159-rep-6cd4655d87-6rgvm    1/1       Running   0          11h
+    default       pvc-40e1b64f-c7ba-11e7-892e-000c29119159-rep-6cd4655d87-h7w9x    1/1       Running   0          11h
+    kube-system   etcd-o-master01                                                  1/1       Running   0          3d
+    kube-system   kube-apiserver-o-master01                                        1/1       Running   0          3d
+    kube-system   kube-controller-manager-o-master01                               1/1       Running   0          3d
+    kube-system   kube-dns-545bc4bfd4-m4ngc                                        3/3       Running   0          3d
+    kube-system   kube-proxy-4ml5l                                                 1/1       Running   0          3d
+    kube-system   kube-proxy-7jlpf                                                 1/1       Running   0          3d
+    kube-system   kube-proxy-cxkpc                                                 1/1       Running   0          3d
+    kube-system   kube-scheduler-o-master01                                        1/1       Running   0          3d
+    kube-system   weave-net-ctfk4                                                  2/2       Running   0          3d
+    kube-system   weave-net-dwszp                                                  2/2       Running   0          3d
+    kube-system   weave-net-pzbb7          
+
+Get the status of running StatefulSets using the following command.
+::
+
+    ubuntu@kubemaster:~$ kubectl get statefulset
+    NAME        DESIRED   CURRENT   AGE
+    couchbase   2         2         11h
+
+Get the status of underlying persistent volume used by Couchbase StatefulSet using the following command.
+::
+
+    ubuntu@kubemaster:~$ kubectl get pvc
+    NAME                         STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+    couchbase-data-couchbase-0   Bound     pvc-16210b06-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+    couchbase-data-couchbase-1   Bound     pvc-40e1b64f-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+
+Get the status of the services using the following command.
+::
+
+    ubuntu@kubemaster:~kubectl get svc
+    ME                                                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
```

Comment:
ME=NAME

Reply:
Done!

Reply:
Not including screenshots as the steps/procedure is obvious to the user and easy to follow. We can discuss if it is required.